src/hotspot/os_cpu/windows_x86/orderAccess_windows_x86.hpp

```diff
@@ -58,13 +58,22 @@ inline void OrderAccess::fence() {
   compiler_barrier();
 }
 
-inline void OrderAccess::cross_modify_fence_impl() {
+inline void OrderAccess::cross_modify_fence_impl()
+#if _MSC_VER >= 1927
+{
+//_serialize() ntrinsic is supported starting from VS2019-16.7.2
```

Comment:
16.7.2 is the earliest I could look at, but it could be supported earlier. Also _MSC_VER of 1927 would start with 16.7.0 so it wouldn't quite be right to test for that if it only come in with 16.7.2.

Typo: ntrinsic

Reply:
I changed to 1928 to be on the safe side